extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/schema/SchemaManagementIntegrator.java

```diff
@@ -99,7 +99,7 @@ public static void recreateDatabase(String name) {
         ServiceRegistry serviceRegistry = holder.sessionFactory.getServiceRegistry();
         SimpleExecutionOptions executionOptions = new SimpleExecutionOptions(serviceRegistry);
         Object schemaGenerationDatabaseAction = executionOptions.getConfigurationValues()
-                .get("javax.persistence.schema-generation.database.action");
+                .get("jakarta.persistence.schema-generation.database.action");
```

Comment:
I didn't change them as I wasn't sure if Hibernate ORM 5.6 was supporting both?

Also should we support both in our code? Or does ORM translate the configuration properties to the new namespace automatically?

Reply:
right, both are supported in theory by ORM 5, but let's aim to switch fully.

Reply:
BTW there's similar things to be done in all other modules.